src/cmd/inspector/main.go

```diff
@@ -66,3 +89,14 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func getCSPTokenFromSecret(ctx context.Context, ns string, secretName string) (error, string) {
+	config, err := secret.NewForConfig(ctrl.GetConfigOrDie())
+	getSecret, err := config.CoreV1().Secrets(ns).Get(ctx, secretName, metav1.GetOptions{})
+	if err != nil {
+		log.Error(err, "Failed to fetch secret")
+		return err, ""
+	}
+	cspApiToken := string(getSecret.Data["accessSecret"])
```

Comment:
Instead of **accessSecret**, it should be secretName.

Reply:
As discussed, its key name is predefined and not controlled by us.